teuthology/suite/util.py

```diff
@@ -453,3 +453,34 @@ def teuthology_schedule(args, verbose, dry_run, log_prefix=''):
         ))
     if not dry_run or (dry_run and verbose > 1):
         subprocess.check_call(args=args)
+
+
+def find_git_parent(project, sha1):
+
+    base_url = config.get('githelper_base_url', 'http://git.ceph.com:8080')
```

Comment:
Ah. I should have noticed this before. Could you put the default for this directly in ``teuthology.config``? Bonus points if you want to add an entry to ``docs/siteconfig.rst``, but I certainly won't block on that.